int8 aruco_id
---
geometry_msgs/PoseStamped coordinates